core-primitives/enclave-metrics/src/lib.rs

```diff
@@ -24,11 +24,27 @@ compile_error!("feature \"std\" and feature \"sgx\" cannot be enabled at the sam
 extern crate sgx_tstd as std;
 
 use codec::{Decode, Encode};
+use std::string::String;
+use substrate_fixed::types::U32F32;
 
-#[derive(Encode, Decode)]
+// copied from ita-exchange-oracle because of cyclic deps
+pub type ExchangeRate = U32F32;
```

Comment:
Can you add a //FIXME or //TODO here? Should be removed after https://github.com/integritee-network/pallets/issues/71